### PR TITLE
release: v1.0.0

### LINKED ISSUE #19: Making a release
I think it could be good to make a release of this package. At INSPIRE we currently use HEAD, and I believe there are no other instances using this package.

### PR DESCRIPTION
**Pull request by kaplun**
Signed-off-by: Samuele Kaplun samuele.kaplun@cern.ch

Closes #19

### CODE REVIEW

File: RELEASE-NOTES.rst [hunk 1]
```diff
@@ -1,36 +1,20 @@
...
+  is found via a synonym or regexp it is no longer thrown away just
+  because the components of the combination are not found in the text.
+- Adds missing `invenio_base` dependency.
 
-What's new
-----------
-
-- Initial public release.
-
-Installation
```
Comment (jirikuncar) [inline RELEASE-NOTES.rst]: do you really want to remove all of it from release notes?

**Comment by jirikuncar** [comment]
@kaplun please update GitHub url in docs and setup.py.

File: RELEASE-NOTES.rst (hunk 1 above)
Comment (kaplun) [inline RELEASE-NOTES.rst]: Ah no. Thanks.

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -1,36 +1,20 @@
...
-   $ pip install invenio-classifier==0.1.0
-
-Documentation
--------------
-
-   http://invenio-classifier.readthedocs.org/en/v0.1.0
-
-Happy hacking and thanks for flying Invenio-Classifier.
-
-| Invenio Development Team
```
Comment (tiborsimko) [inline RELEASE-NOTES.rst]: You may want to amend the maintainer signature in the release notes, setup.py, readme, contributing, docs...

Comment (kaplun) [inline RELEASE-NOTES.rst]: This is a good point. Wasn't entirely sure about what to put it. But I guess I shall.

**Comment by kaplun** [comment]
@tiborsimko @jirikuncar I think I have applied all requests.

**Comment by jirikuncar** [comment]
@kaplun normally we use following commit messages during the release process: `release: v{__version__}` and `version: post-release bump`.

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -109,5 +109,5 @@ Before you submit a pull request, check that it meets these guidelines:
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring.
 3. The pull request should work for Python 2.7, 3.3, 3.4 and 3.5. Check
```
Comment (jirikuncar) [inline CONTRIBUTING.rst]: are you sure about `2.7, 3.3, 3.4 and 3.5`?

Comment (kaplun) [inline CONTRIBUTING.rst]: Right. It was the case in the past but indeed due to #21 this is no longer a requirement.

**Pull request merged**